mathics/builtin/specialfns/orthogonal.py

```diff
@@ -259,8 +260,8 @@ class SphericalHarmonicY(MPMathFunction):
     """
     <url>:Spherical Harmonic https://mathworld.wolfram.com/SphericalHarmonic.html</url> (<url>:mpmath: https://mpmath.org/doc/current/functions/orthogonal.html#mpmath.sperharm</url>, <url>:WMA: https://reference.wolfram.com/language/ref/SphericalHarmonicY.html</url>)
     <dl>
-      <dt>'SphericalHarmonicY'[$l$, $m$, $theta$, $phi$]
-      <dd>returns the spherical harmonic function Y_$l$^$m$(theta, phi).
+      <dt>'SphericalHarmonicY'[$l$, $m$, $\theta$, $\phi$]
+      <dd>returns the spherical harmonic function $Y_l^m(\theta, \phi)$.
```

Comment:
Careful - you need to escape `\`, e.g.  `\\theta` and `\\phi` here and in the line above.  Remember to _look_ at the results in Django.

Reply:
I forgot to mark the string as a raw string...

Reply:
In general, I find easier to read raw docstrings.